### PR TITLE
feat(RELEASE-1144): push content sets to Pyxis

### PR DESCRIPTION
**Pull request by mmalina**
The `upload_rpm_manifest` script was renamed to
`upload_rpm_data`.

On top of the rpm manifest, it will now also update the ContainerImage.content_sets field in Pyxis.

The content sets is a list of unique strings
taken from the repository_id qualifier of purl
entries.

I tested the script by running it against several images in stage Pyxis. I hit different scenarios (no content sets in pyxis, then saved and skipped, and also no content sets in the sbom) and it all seemed to work.

### CODE REVIEW

File: pyxis/upload_rpm_data.py [hunk 1]
```diff
@@ -55,29 +60,44 @@ def upload_container_rpm_manifest_with_retry(
     raise last_err
 
 
-def upload_container_rpm_manifest(graphql_api: str, image_id: str, sbom_path: str):
+def upload_container_rpm_data(graphql_api: str, image_id: str, sbom_path: str):
     """Upload a Container Image RPM Manifest to Pyxis"""
```
Comment (johnbieren) [inline pyxis/upload_rpm_data.py:64]: This should probably be updated

**Review by johnbieren — COMMENTED** [review]
Unfortunate that github can't render the test file diff, but I diff'ed it locally and it seems sane. One nit pick

**Comment by mmalina** [comment]
> Unfortunate that github can't render the test file diff, but I diff'ed it locally and it seems sane. One nit pick

Yeah, it's weird. It did show the diff properly for the main py file, but not for the test file. Maybe too many changes?

File: pyxis/upload_rpm_data.py (hunk 1 above)
Comment (mmalina) [reply pyxis/upload_rpm_data.py]: Good catch, fixed.

**Review by johnbieren — APPROVED** [review]

**Pull request merged**